LightningNetwork.md

```diff
@@ -0,0 +1,93 @@
+# Lightning Network and BTCPay
+
+After deploying BTCPay Server, as a merchant, you may want to experiment with an innovative second-layer payment system built on top of Bitcoin protocol - the [Lightning Network](https://en.bitcoin.it/wiki/Lightning_Network).
+
+This guide will show you how to set up your Lightning Network node in BTCPay and guide you through some basics.
+
+<strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. Do not put the money you can't afford to lose. There is a high risk of you losing the money. </strong>
+
+Take time to familiarize yourself with the risk. <strong>There's no backup for LND or c-lightning keys in BTCPay. Your keys are in a hot-wallet</strong>. This means :
+
+1. If you erase your BTCPay Server or your machine crashes - you lose all the funds.
+2. If your server gets hacked - a hacker can take all of your funds by accessing your keys.
+
+While the keys from your Lightning Network don't have a backup and someone can steal them theoretically, your on-chain Bitcoin funds are safe and are never uploaded on the server.
+
+As the technology matures and develops, things like a proper backup will be easier to implement in BTCPay.
+
+BTCPay currently offers two implementations of the Lightning Network:
+
+* [LND](https://github.com/lightningnetwork/lnd)
+* [c-lightning](https://github.com/ElementsProject/lightning)
+
+## Choosing the Lightning Network implementation
+
+On the installation, you'll have the option to choose the implementation. For [web-interface installations](https://medium.com/@BtcpayServer/launch-btcpay-server-via-web-interface-and-deploy-full-bitcoin-node-lnd-in-less-than-a-minute-dc8bc6f06a3), you can simply select the implementation from the drop-down menu. For [docker](https://github.com/btcpayserver/btcpayserver-docker) you need to :
+
+```
+sudo su -
+cd btcpayserver-docker
+export BTCPAYGEN_LIGHTNING="implementationgoeshere"
+. ./btcpay-setup.sh -i
+```
+
+For c-lightning use `export BTCPAYGEN_LIGHTNING="clightning"`
+For LND use `export BTCPAYGEN_LIGHTNING="lnd"`
+
+To begin using Lightning, your blockchain needs to be fully synced.
+
+## Connecting your internal Lightning Node in BTCPay
+
+Regardless of the implementation (c-lightning or LND) you've decided to use, the process of connecting your internal Lightning Node in BTCPay Server is the same.
+
+1. If you do not have a store, create one. 
+2. Store Settings > General Settings > Lightning Network Experimental (located at the bottom of the page, scroll)
+3. Under Crypto tab, select cryptocurrency > Modify.
+4. At the next page, at the bottom under "connection string", click on the "click here" link. Your node information will be automatically added.
+5. Enable. Submit.
+6. Test Connection.
+
+![LightningNetworkSettup1](img/LLightningNetworkNodeSetup1.jpg)
```

Comment:
Typo, should be `img/LightningNetworkNodeSetup1.jpg`

Reply:
Fixed, thanks.